log/loggers.go

```diff
@@ -19,6 +20,9 @@ type LoggerInfo struct {
 	// If 'showTime' is true, it will print the time for each line displayed
 	// by the logger.
 	ShowTime bool
+	// If 'AbsoluteFilePath' is true, it will print the absolute file path
+	// instead of the base.
+	AbsoluteFilePath bool
```

Comment:
Don't you want to add that to `ParseEnv` to enable it with `ABSOLUTE_FILEPATH=true`?

Reply:
or `DEBUG_FILEPATH="true"`?

Reply:
#584

Reply:
Erf I just wrote the same commit

Reply:
;)